scripts/sv/runWholePipeline.sh

```diff
@@ -34,6 +35,16 @@ KMER_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.kmers/')
 ALTS_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.alts/')
 REF_TWOBIT=$(echo "${REF_FASTA}" | sed 's/.fasta$/.2bit/')
 
+# extract any extra arguments to StructuralVariationDiscoveryPipelineSpark
+shift 6
+SV_ARGS=${*:-${SV_ARGS:-""}}
+# expand any local variables passed as strings (e.g. PROJECT_OUTPUT_DIR)
+eval "SV_ARGS=\"${SV_ARGS}\""
+
+# Choose NUM_EXECUTORS = 2 * NUM_WORKERS
+NUM_WORKERS=$(gcloud compute instances list --filter="name ~ ${CLUSTER_NAME}-[sw].*" | grep RUNNING | wc -l)
```

Comment:
That would eliminate the warning.  I think the warning itself is actually a bug (it corresponds to passing the NAME argument, not using a filter). I've avoided listing all instances and using grep because that corresponds to potentially downloading a large number of instances, whereas this in principle might filter before download, saving data. If the warning is disconcerting, I can make the fix you suggest and then revert once google gets their act together on this.